.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do you add Swing?
The console app would be enough.

Also please remove .class files. 
You can add .gitignore.